docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -73,13 +71,69 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>1236</td>
```

Comment:
This seems too low level to include, in my opinion

Reply:
I will remove the listed fixed issues that are low level code changes, since they don't appear to yield direct user benefit.